### ISSUE TITLE
Firefox: Extensions stops working after leaving site/no save button in configuration pop-up

### ISSUE THREAD

**Issue by mr-aufziehvogel** [mrinc/FB-Hide-Recommendations-and-Reels#64]
Hi!

I'm using the latest version on Firefox. I get the initial configuration wizard, save my configuration and it works, but the next time I open Facebook, everything looks as usual. When I open the configuration pop-up, the options are reset and the save button doesn't show up, either.

Laden fehlgeschlagen für das <script> mit der Quelle "moz-extension://851f3a02-2a62-4873-8128-fa10783c613f/tailwindcss.js". popup.html:7:31
Uncaught TypeError: r.storage.sync.get(...) is undefined
    initPopupConfig moz-extension://851f3a02-2a62-4873-8128-fa10783c613f/popup.js:1
    initPopupConfig moz-extension://851f3a02-2a62-4873-8128-fa10783c613f/popup.js:1
    <anonymous> moz-extension://851f3a02-2a62-4873-8128-fa10783c613f/popup.js:1
    initPopupHome moz-extension://851f3a02-2a62-4873-8128-fa10783c613f/popup.js:1
    onload moz-extension://851f3a02-2a62-4873-8128-fa10783c613f/popup.js:1
    EventHandlerNonNull* moz-extension://851f3a02-2a62-4873-8128-fa10783c613f/popup.js:1
    <anonymous> moz-extension://851f3a02-2a62-4873-8128-fa10783c613f/popup.js:1
    <anonymous> moz-extension://851f3a02-2a62-4873-8128-fa10783c613f/popup.js:1
popup.js:1:30928

That's the output I get in the console. I hope that helps.

Thanks for your work!

**Comment by mrinc** [comment]
There isn't a save button on the config page :)   
It saves as soon as you check/uncheck - and then refreshes the page when you navigate out... might be a good idea to add one now due to the flow of the initial wizard.  

But that config is undefined is defs an error ....

**Comment by mr-aufziehvogel** [comment]
Oh, I see :D 
Changing the settings pop up works anyway. It just doesn't last. I wondered, if other extensions interfere with it, but that doesn't seem to be the case

**Comment by mrinc** [comment]
:D Legacy config popup.  
When we added the intro popup, we added the button (the intro button also doesn't do anything but close the window) - but naturally feels weird now, so will look at that.   

So it does change, but then gets reverted ...mmmm   

I can replicate the issue.  
So the config is saved, but not being reset so visually looks like it is disabled. Will inspect

**Comment by mrinc** [comment]
@mr-aufziehvogel  - please test with changes: https://github.com/mrinc/FB-Hide-Recommendations-and-Reels/releases/tag/v1.26.27-rc0

**Comment by mr-aufziehvogel** [comment]
I had to install Nightly, since it's not validated. I don't know, if it makes a difference.
It works more consistently now - but only after refreshing the page. Otherwise it won't do anything

**Comment by mrinc** [comment]
if you go into extension settings. enable dev mode. you can load an unpacked extension ( download zip, extract to directory and load that way)
should do the job :)

if its first install, you would need to refresh fb once for it to load.
and any changes to settings, once settings popup closes, should refresh the page for you.

can you confirm more on the refresh issue? 


sponsored content / ads?
we don't/can't hide ads but do have some code that sometimes will hide ads if the ads url is visible - so you have to click the ... icon for it to render the ads url.
they make that extremely difficult - so it's not an advertised feature of this plugin :) 

sponsored content ... mmm, possible they changed the text for sponsored content for your lang - can you confirm what it says and we can add it?
